src/Compilers/Core/Portable/System/Reflection/Metadata/ILOpCode.cs

```diff
@@ -204,6 +204,9 @@ enum ILOpCode : ushort
         Leave_s = 0xde,
         Stind_i = 0xdf,
         Conv_u = 0xe0,
+        // LoadMethodTokenAsInteger_i4 is a pseudo-opcode used to trick the metadata writer into putting a method token value into a ldc_i4.
+        // Its value must fit in one byte (because it will be overwritten with Ldc_i4) and must not collide with a real opcode.
+        LoadMethodTokenAsInteger_i4 = 0xef,
```

Comment:
0xef is the greatest allowed value, and so is the furthest from expected future conflicts.